site/_posts/2020-09-22-manageiq-jansa-ga-announcement.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "ManageIQ Jansa GA announcement"
+author: Fryguy
+date: 2020-09-22
+tags: releases announcements
+comments: true
+published: true
+---
+
+On behalf of the ManageIQ team, I'm happy to announce the release of **ManageIQ Jansa**!  This release is named after Czech chess grandmaster [Vlastimil_Jansa](https://en.wikipedia.org/wiki/Vlastimil_Jansa).  You can download the Jansa-1 release on our [downloads page](/downloads).
```

Comment:
minor, but the chess player's name does not contain an underscore (although the link does)